content/contributors/_index.md

```diff
@@ -101,6 +101,12 @@ The Cloud Native Computing Foundation projects are listed [below](projects/), to
 |                  [Devfile](projects/#devfile)                  |           Devfile           |          Go          |
 |                 [Kyverno](projects/#kyverno)               | Kubernetes Native Policy Management |          Go          |
 
+### Non-code Projects
+
+|                          Project Name                          |            Focus            |   Primary Language   |
+| :------------------------------------------------------------: | :-------------------------: | :------------------: |
+|             [Cloud Native Glossary](projects/#glossary)        |           Definitions       |        Markdown      |     
```

Comment:
Here's the correct link so that it jumps directly to the glossary

```suggestion
|             [Cloud Native Glossary](projects/#cloud-native-glossary)        |           Definitions       |        Markdown      |     
```